tests/e2e/tests/subscription.spec.js

```diff
@@ -0,0 +1,204 @@
+const { test, expect } = require('@playwright/test');
```

Comment:
JFYI
Better to make page object models with getters, so all your elements become reusable.
Consider adding test ids to your UI elements to maintain easy and clean getters or your getter might look like mine real ones 
```
  return this.getFormControl('center')
      .find(`input[aria-valuenow="${centerFrequency}"]`)
      .closest('.frequency-row')
      .find('mat-icon')
      .contains('link');
}
```
This happens when you don't want to add element id's